uvtools/tests/test_dspec.py

```diff
@@ -797,6 +808,17 @@ def get_snr(clean, fftax=1, avgax=0, modes=[2, 20]):
 
     #check that fringe rate filter model gives similar results to delay filter.
     nt.assert_true(np.all(np.isclose(mdl1[~f],mdl5[~f], rtol=1e-2)))
+
+    #perform some sanity checks on handling of nans in dft_leastsq
+    mdl, res, info = dspec.fourier_filter(x=freqs, data=d[0], wgts=w[0], filter_centers=[0.],
```

Comment:
I think it could be made a little more clear that this is testing that nans are handled correctly. I understand that the default setting for a dft-based filter is to set the fundamental period to `np.nan`, but this may be confusing if there are other people who contribute to these tools in the future. I suppose this isn't essential, but would be good to implement in the future if we're planning on raising the standards to which we hold the code in this repo.

Reply:
I've added additional documentation to explain this.